test/functional/apps/discover/_saved_queries.ts

```diff
@@ -134,10 +133,6 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await savedQueryManagementComponent.saveNewQueryWithNameError('OkResponse');
       });
 
-      it('does not allow saving a query with leading or trailing whitespace in the name', async () => {
```

Comment:
Note: this is no longer needed as leading and trailing whitespace is automatically trimmed in UI